pkg/model/serializer.go

```diff
@@ -457,17 +457,15 @@ func (s *Secret) UnmarshalYAML(unmarshal func(interface{}) error) error {
 		}
 	}
 
+	// if the secret is not formatted correctly we return an empty secret (i.e. empty LocalPath and RemotePath)
+	// and we rely on the secret validation to return the appropriate error to the user
 	if len(parts) < 2 || len(parts) > 3 {
```

Comment:
```suggestion
	if len(parts) != 2  {
```

Reply:
if we change this, we have unreachable conditions below, where it checks for `== 3`